CONTRIBUTING.md

```diff
@@ -81,9 +81,15 @@ Editable notebook, to run the random tweeted configurations is made available us
 Code coverage (how much of our code is actually seen by the (linux) unit tests) is tested using Codecov, a report is visible on https://app.codecov.io/gh/pybamm-team/BattBot/.
 
 ## Pre-commit checks
-1. Style `$ flake8`
-2. Tests `$ coverage run --concurrency=multiprocessing -m unittest` or `$ python -m unittest`
+1. Style `$ flake8 --max-line-length=89`
+2.
```

Comment:
Here we can do something like -

2. Tests - `$ python -m unittest -v` (if you have completed the pre-installation process) or `$ python -m unittest -v discover ./test/without_keys` (if you have haven't completed the pre-installation process)

I think we can get rid of the `coverage` commands